packages/e2e-tests/helpers/utils.ts

```diff
@@ -179,3 +179,53 @@ export async function resetTestUser(email: string) {
     },
   });
 }
+
+export async function cloneTestRecording(recordingId: string): Promise<string> {
+  const variables = { recordingId, secret: process.env.AUTOMATED_TEST_SECRET };
```

Comment:
Should we throw if `process.env.AUTOMATED_TEST_SECRET` is undefined? Might be clearer that way that it's an environmental/configuration issue.

Reply:
Done. Though, if we only had public recording this is not required. Note to future PR visitors. 👋

Reply:
Oh, sorry! I meant for my comment to be about `process.env.AUTHENTICATED_TESTS_WORKSPACE_API_KEY`, since (I think) the API clone request would fail if that param was missing.

Reply:
Oh, sorry! I meant for my comment to be about `process.env.AUTHENTICATED_TESTS_WORKSPACE_API_KEY`, since (I think) the API clone request would fail if that param was missing.

Reply:
Right, even though I understood you meant this env I relied too much copilot haha. Fixing. Same note applies though, that's an optional header. We can skip whole header if public or pass this if it's private. We have both so passing this to public recording won't have a problem.